packages/types/src/type/UncleEntryItem.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import Enum from '../codec/Enum';
+
+/**
+ * @name UncleEntryItem
+ */
+export default class UncleEntryItem extends Enum {
```

Comment:
This encoding seems incorrect: https://github.com/paritytech/substrate/blob/c3f6617d2b9043c78f8e754e1856fddc29bc6fa1/srml/authorship/src/lib.rs#L167

InclusionHeight is a BlockNumber, and Uncle is a `Tuple(Hash, Option<AccountId>)`